packages/dds/tree/src/feature-libraries/chunked-forest/codec/format.ts

```diff
@@ -64,6 +64,12 @@ export const EncodedFieldShape = Type.Tuple([
 
 export type EncodedFieldShape = Static<typeof EncodedFieldShape>;
 
+/**
+ * Content of the encoded field is an array of chunk reference ids, one for each chunk. Each chunk in the field is
+ * encoded separately and is referenced by its reference id.
+ */
+export const EncodedIncrementalShape = Type.Literal(0);
```

Comment:
This PR comment https://github.com/microsoft/FluidFramework/pull/24949/files#r2232221650 probably would be better placed here.